package.json

```diff
@@ -1,8 +1,11 @@
 {
   "name": "is-relative-path",
-  "version": "1.0.2",
+  "version": "2.0.0",
   "description": "Whether or not a given path is relative",
   "main": "index.js",
+  "engines": {
+    "node": ">0.11.2"
```

Comment:
I made a mistake. The correct value is `">=0.11.2"` because [`path.isAbsolute()`](https://nodejs.org/api/path.html#path_path_isabsolute_path) was added in v0.11.2.